### PR TITLE
Experimental Antora framework for Vulkan Documentation Project demo

### PR DESCRIPTION
**Pull request by oddhack**
Scripts to modify asciidoctor markup as needed into Antora-compliant form, and Antora navigation files not found in the asciidoctor source.

See https://github.com/KhronosGroup/Vulkan-Site for the Antora playbook used to generate the Vulkan Documentation Project site, and https://registry.khronos.org/vulkan/site/spec/latest/index.html for the most recently published version of this demo.

### CODE REVIEW

File: antora/modules/ROOT/nav.adoc [hunk 1]
```diff
@@ -0,0 +1,66 @@
+# Copyright 2022-2023 The Khronos Group Inc.
+# SPDX-License-Identifier: CC-BY-4.0
+
+:chapters:
+
+* xref:index.adoc[Vulkan Guide]
+* Logistics Overview
+** xref:{chapters}what_is_vulkan.adoc[]
```
Comment (spencer-lunarg) [inline antora/modules/ROOT/nav.adoc:8]: Does this mean we need to add a new line here as well whenever a new file/chapter is added?

Comment (oddhack) [reply antora/modules/ROOT/nav.adoc:8]: Yes. This is obviously amenable to massaging out of the toplevel README.adoc but some scripting will be required. At the rate chapters are being added it doesn't seem like a big burden though we might want to put a comment in the README to remind people?

Comment (spencer-lunarg) [reply antora/modules/ROOT/nav.adoc:8]: agree not the worst thing ever, but yes, a small blurb in `CONTRIBUTING.adoc` to remind people to add it here (and maybe a quick "why we are using Antora" for those who can't find this PR)

Comment (oddhack) [reply antora/modules/ROOT/nav.adoc:8]: Done in CONTRIBUTING.adoc.

**Comment by oddhack** [comment]
@spencer-lunarg if the CONTRIBUTING comments are OK with you, can we go ahead and get this merged?

**Review by spencer-lunarg — APPROVED** [review]

**Pull request merged**